src/main/java/racingcar/ResultView.java

```diff
@@ -6,7 +6,7 @@ public class ResultView {
     public void showCurrentState(ArrayList<Car> cars) {
         for (Car car : cars) {
             String dashes = "-".repeat(car.currentPosition());
-            System.out.println(dashes);
+            System.out.println(car.getName() + " : " + dashes);
```

Comment:
```suggestion
            System.out.println(car.nameAppendTo(dashes));
```

좋지는 않은것 같지만, get 네이밍을 안쓰기 위해 억지로? 해보면 이런 메소드가 나올수도 있겠네요 ㅋ
(객체에게 메시지를 전달해보는 훈련의 하나의 과정으로요 ㅎ)